app/utils/IncidentP5Helper.scala

```diff
@@ -17,14 +17,18 @@
 package utils
 
 import generated.IncidentType03
+import play.api.Logging
 import play.api.i18n.Messages
 import uk.gov.hmrc.govukfrontend.views.viewmodels.summarylist.SummaryListRow
 import viewModels.sections.Section.StaticSection
 
 class IncidentP5Helper(
   data: IncidentType03
 )(implicit messages: Messages)
-    extends DeparturesP5MessageHelper {
+    extends DeparturesP5MessageHelper
+    with Logging {
+
+  val displayIndex = data.sequenceNumber + 1
```

Comment:
I can't see any tests for the IDs? I'm intrigued to see what `(data.sequenceNumber: String) + 1` will return

Reply:
@karahanozturk Fair. Sequence numbers start from 1, so the + 1 is not needed anyway. As long as we remember to unit test the IDs when actions get added it's all good.